client/components/promo-section/index.tsx

```diff
@@ -12,6 +12,7 @@ import PromoCardCta, { Props as PromoCardCtaProps } from './promo-card/cta';
 interface PromoSectionCardProps extends PromoCardProps {
 	body: string;
 	cta?: PromoCardCtaProps;
```

Comment:
Is there a mistake here?

```suggestion
	cta: PromoCardCtaProps['cta'];
```